<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>650</width>
      <height>500</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>650</width>
      <height>500</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>650</width>
      <height>500</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="m_singleWidgetButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>69</y>
      <width>93</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> color: #FFFFFF;background-color: #F78F4F;
</string>
    </property>
    <property name="text">
     <string>单字输入</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_multiWidgetButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>93</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>多字输入</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
